docs/UserGuide.md

```diff
@@ -206,117 +174,120 @@ Date:   Fri Feb 9 19:13:13 2018 +0800
+Repository's Location | The `GitHub URL` or `Disk Path` to the git repository e.g., `https://github.com/foo/bar.git` or `C:\Users\user\Desktop\GitHub\foo\bar`
+Branch | The branch to analyze in the target repository e.g., `master` Default: the default branch of the repo
+[Optional] File formats<sup>*</sup> | The file extensions to analyze. Default: `adoc;cs;css;fxml;gradle;html;java;js;json;jsp;md;py;tag;xml`
+[Optional] Ignore Glob List<sup>*</sup> | The list of file path globs to ignore during analysis for each author. e.g., `test/**;temp/**`
+[Optional] Ignore standalone config | To ignore the standalone config file (if any) in target repository, enter **`yes`**. If the cell is empty, the standalone config file in the repo will take precedence over configurations provided in the csv files.
+[Optional] Ignore Commit List<sup>*</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash.
+
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+
+#### `author-config.csv`
+
+Optionally, you can use a `author-config.csv` (which should be in the same directory as `repo-config.csv` file) to provide more details about the authors to analyze ([example](author-config.csv)). It should contain the following columns:
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | Same as `repo-config.csv`
+Branch | The branch to analyze for this author
+Author's GitHub ID | GitHub username of the target author e.g., `JohnDoe`
+[Optional] Author's Display Name | The name to display for the author. Default: author's GitHub username.
+[Optional] Author's Git Author Name<sup>*</sup> | Detailed explanation below
+[Optional] Ignore Glob List<sup>*</sup> | Files to ignore for this author, in addition to files ignored by the patterns specified in `repo-config.csv`
+
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+
+If `author-config.csv` is not given and the repo has not provide author details in a standalone config file, all the authors of the repositories will be analyzed.
+
+<hr>
+
+## Analyzing Multiple Repos
+
+This section assumes that you have read the earlier sections of the user guide.
+
+The simplest way to analyze multiple repos in one go is to use the `-repos` parameter in place of the `-repo` parameter when running RepoSense.
+* Format : `java -jar RepoSense.jar -repos REPO_LIST` <br>
+* Example: `java -jar RepoSense.jar -repos https://github.com/reposense/RepoSense.git c:/myRepose/foo/bar` analyzes the two specified repos (one remote, one local) and generates one report containing details of both.
+
+Alternatively, you can use csv config files to customize the analysis as before while specifying multiple repos to analyze.
+* `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
```

Comment:
The hyperlink to the example, ([example](repo-config.csv)), was added here but not found in its own, `repo-config.csv`, section.